pkg/reconciler/openshift/testing.go

```diff
@@ -0,0 +1,86 @@
+package openshift
+
```

Comment:
You may want to rename this file to openshift_test.go so that it only gets built by `go test`.

Reply:
renamed the file

Reply:
I should point out that this is pretty much a zero-cost thing, the logic from that file won't end up in the final binary.

And naming it "testing.go" is actually pretty common, for examples https://github.com/hashicorp/nomad/blob/master/client/testing.go https://github.com/hashicorp/nomad/blob/master/command/agent/consul/testing.go https://github.com/kubernetes/kubernetes/blob/master/pkg/volume/testing/testing.go